disks-and-volumes.md

```diff
@@ -0,0 +1,315 @@
+# Volumes and Disks
+
+## Disks
+
+Like all other vm devices a `spec.domain.devices.disks` element has a mandatory
+`name`. Furhter it has a mandatory `volumeName` entry which references a volume
+inside `spec.volumes`.
+
+A disk can be made accessible via four different types:
+
+ * disk
+ * cdrom
+ * floppy
+ * lun
+
+### lun
+
+A `lun` disk will expose the volume as a LUN device to the vm.
```

Comment:
Could you link to a definition of `lun` - especially in contrast to disks.

Reply:
Not sure where to link. The difference is what you see in the guest. Do you mean linking to the api-reference?

Reply:
IIUIC then the diffrerence between lun and disk is that with lun more scsi calls are passed through disk is somewhat filtering it - that's at least what I get from the libvirt docs.
Is there any other difference I am missing?

Reply:
I can add to the api-reference docs that this allows generic iscsi command passthrough.

Reply:
Yep, that would work for me, or even just a sentence here differentiating lun from disk. At least some hint how they differ :)

Reply:
done